src/components/home/cluster_dashboard_item.js

```diff
@@ -70,6 +71,19 @@ class ClusterDashboardItem extends React.Component {
     return 0;
   }
 
+  /**
+   * Returns true if the cluster is younger than 30 days
+   */
+  clusterYoungerThan30Days() {
+    var age = Math.abs(
+      moment(this.props.cluster.create_date)
+        .utc()
+        .diff(moment().utc()) / 1000
+    );
+
+    return age < 30 * 24 * 60 * 60;
```

Comment:
small perf improvement: calculate this in a class var as constant to avoid do the calculation every call :)